templates/galaxy/config/object_store_conf.xml.j2

```diff
@@ -246,38 +217,73 @@ The storage consists of 16 Gen6 type A200 storage nodes and 16 Gen5 type X410 no
             <files_dir path="/data/dnb11/galaxy_db/files"/>
             <extra_dir type="job_work" path="/data/jwd07/main"/>
         </backend>
-        <backend id="secondary" type="disk" weight="0" store_by="id">
-            <files_dir path="/data/0/galaxy_db/files"/>
-            <extra_dir type="temp" path="/data/2/galaxy_db/tmp"/>
-            <extra_dir type="job_work" path="/data/jwd/main"/>
-        </backend>
-        <backend id="dataplant01" type="disk" weight="0" store_by="uuid">
+        <backend id="dataplant01" type="disk" weight="0" store_by="uuid" device="dp01">
             <files_dir path="/data/dp01/galaxy_db/files"/>
             <extra_dir type="temp" path="/data/jwd/object_store_temp/dataplant01"/>
             <extra_dir type="job_work" path="/data/jwd/main"/>
         </backend>
-
-        <!--
-        <backend type="generic_s3" id="s3_netapp01" allow_selection="false" weight="0" store_by="uuid" name="Short term storage for method development">
-            <description>S3 based objetct storage matained by the compute center of the University of Freiburg.
-                         This storage is purged weekly and so it is only appropriate for short term methods development and such.
-                         The rapid deletion of stored data enables us to provide this storage without a quota.
-                         More information about our storage offerings can be found [here](https://galaxyproject.org/eu/).
+        
+        <!-- also called Scratch storage -->
+        <backend type="generic_s3" id="s3_scratch_netapp01" allow_selection="true" weight="0" store_by="uuid" device="scratch" name="Short term storage for e.g. method development">
+            <description>S3-based object storage is maintained by the compute center of the University of Freiburg.
+                         Data on this storage, also called scratch storage, is deleted if it is older than two months (60 days), so it is only appropriate for short-term methods development.
+                         The rapid deletion of stored data enables us to provide this storage without a quota. This storage is not backed up.
+                         More information about our storage offerings is documented at [https://galaxyproject.org/eu/storage/](https://galaxyproject.org/eu/storage/).
             </description>
+            <quota enabled="false" source="scratch" />            
```

Comment:
```suggestion
            <quota source="scratch" />            
```

That should be enough, once this is merged I can create 2 new quotas and I will see the source label - I have tested this over the weekend. 2 new quotas for registered and unregistered users.